frontend/src/SDKInitialize.tsx

```diff
@@ -24,7 +24,11 @@ const config: React.ComponentProps<typeof AppInitSDK>['configurations'] = {
   wsAppSettings: () => Promise.resolve({ host: '', subProtocols: [] }),
 };
 
-const SDKInitialize: React.FC = ({ children }) => {
+type SDKInitializeProps = {
+  children?: React.ReactNode;
```

Comment:
I know the original type was optional -- are we sure we don't want to just go all in and make them required when needed? (this applies to all uses of `children` prop)